Help/.coafile

```diff
@@ -0,0 +1,4 @@
+[cli]
+bears = SpaceConsistencyBear
+files = Getting_In_Touch.rst
+use_spaces = True
```

Comment:
don't need this. Why have you added a `.coafile` here? There is already one in the project that has the `SpaceConsistencyBear`

Reply:
@anctartica I did'nt added this file, I just ran the coala's space consistency bear on Getting_In_Touch.rst file and this .coafile generated automatically

Reply:
please remove it from the commit